src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const propertiesObject = {};
+
+  sourceString
+    .split(';')
+    .map(line => line.split(':').map(element => element.trim()))
```

Comment:
Calling map() method but not using its result. If you just need to iterate over an array and perform side effects, use forEach(). If you want to transform an array into a new array, use map() and assign its result to a variable or return it.